src/test_utils/mod.rs

```diff
@@ -1,19 +1,20 @@
 use crate::{
     abe_policy::{DimensionBuilder, EncryptionHint, Policy},
-    Error,
+    api::Covercrypt,
+    Error, MasterPublicKey, MasterSecretKey,
 };
 
 // pub mod non_regression;
 
-/// Creates the test policy.
-pub fn policy() -> Result<Policy, Error> {
+/// Creates the MSK object used in tests.
+pub fn msk() -> Result<(MasterSecretKey, MasterPublicKey, Covercrypt), Error> {
     let sec_level = DimensionBuilder::new(
         "Security Level",
         vec![
             ("Protected", EncryptionHint::Classic),
             ("Low Secret", EncryptionHint::Classic),
             ("Medium Secret", EncryptionHint::Classic),
-            ("High Secret", EncryptionHint::Classic),
+            ("Confidential", EncryptionHint::Classic),
```

Comment:
I wouldn't change that, rather change the test requiring a `Confidential` attribute ?